src/move/econia/sources/user.move

```diff
@@ -409,6 +409,59 @@ module econia::user {
         size: u64
     }
 
+    /// Table keys are market account IDs.
+    struct MarketEventHandles has key {
+        fill_events: Tablist<u128, EventHandle<FillEvent>>,
+        place_limit_order_events:
+            Tablist<u128, EventHandle<PlaceLimitOrderEvent>>,
+        place_market_order_events:
+            Tablist<u128, EventHandle<PlaceMarketOrderEvent>>
+    }
+
+    struct FillEvent has copy, drop, store {
+        market_id: u64,
+        size: u64,
+        price: u64,
+        maker_side: bool,
+        maker: address,
+        maker_custodian_id: u64,
+        maker_order_id: u128,
+        taker: address,
+        taker_custodian_id: u64,
+        taker_order_id: u128
+    }
```

Comment:
Consider including information about fees here? 

(I said this once [here](https://github.com/econia-labs/econia/pull/321#discussion_r1252595147), but I don't think you ever addressed it.)

Reply:
After your previous comment about fees I incorporated the fee information into associated `PlaceLimitOrderEvent`, `PlaceMarketOrderEvent`, or `PlaceSwapOrderEvent`, but per https://github.com/econia-labs/econia/pull/321#issuecomment-1626162868 I have incorporated into the proposed `TakerFillSummaryEvent` in an effort to isolate fee information.

Since the matching engine computes fees using integer division, fees are assessed after all fills for a trade have been executed, to amortize rounding errors. Hence there is no explicit fee associated with any one fill. Fees are also only assessed on the taker side of a trade, so I'm proposing the `TakerFeeSummaryEvent`: this provides information about the amount of quote and base assets traded away/received by the taker, the amount paid in fees, and the number of associated fills. Hence I also added a sequence number to each fill within a trade.

Reply:
Per discussion with @qdrs for a concrete example of fills:

Two fills of equal size constitute a trade such that the matching engine assesses 201 indivisible quote subunits of fees. If fees were reported for each `FillEvent`, integer truncation would result in an output 100 subunits of fees per, even though the matching engine assesses 201. Hence a 1 subunit accounting error. This is resolved by the summary fee information in a `TakerFillSummaryEvent`